### DISCUSSION TITLE
Partials file name error

### DISCUSSION THREAD

**Discussion by rsevs3** [pradyunsg/furo#726] [Potential Issue] [answered]
When I try to build using sphinx-build I am getting the following error:

```
Theme error:
An error happened in rendering the page discussion/index-discussion.
Reason: TemplateNotFound('_partials/_head_css_variables.html')
```

If i rename the theme/furo/partials folder to _partials, the build runs successfully.

**Comment by pradyunsg** [comment, marked as answer]
I can't reproduce this and given that no one else has reported this issue, I'm gonna say this doesn't warrant additional investigation at this time.
